### ISSUE TITLE
udn.com : nuisance:

### ISSUE THREAD

**Issue by weijen6** [uBlockOrigin/uAssets#17978]
### Prerequisites

- [X] This is not a support issue or a question. For support, questions, or help, visit [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/).
- [X] I performed a [cursory search of the issue tracker](https://github.com/uBlockOrigin/uAssets/issues?q=is%3Aissue) to avoid opening a duplicate issue.
- [X] I am running the latest version of uBlock Origin (uBO).

### I tried to reproduce the issue when...

- [X] uBO is the only extension.
- [X] uBO uses default lists and settings.
- [X] using a new, unmodified browser profile.

### Description

Ask if you are an adult?
udn.com

### URL(s) where the issue occurs.

https://udn.com/news/story/7317/7148204

### uBO version

1.49.2

### Browser name and version

Chrome 112.0.5615.138

### Settings

default

**Comment by gorhill** [comment]
I don't know if it's something one of the Annoyances list would add, but here it is:

    udn.com##.udn-overlay
    udn.com##.classification-wrapper
    udn.com##body:style(overflow: unset !important)

**Comment by ryanbr** [comment]
Is it an age check? don't currently filter these, uBO annoyances maybe

**Comment by okiehsch** [comment]
> Is it an age check?

Yes, it's ok if you don' fix those kind of issues, I wasn't sure that's why I asked.

I don't think we ever did, @gorhill what do you think?
We never added filters for age checks as far as I know.

**Comment by weijen6** [comment]
Sorry to interrupt, I checked the FilterLists, there is an Age Check rule in the AdGuard Annoyances filter, such as manga.club###adultConfirmModal

**Comment by okiehsch** [comment]
@Yuki2718 something for `AdGuard Annoyances`

**Comment by gorhill** [comment]
> We never added filters for age checks as far as I know.

Let's keep it that way, it's easily dismissable and not sure it's universally agreed it's a nuisance.
